libs/application/api/history/src/lib/history.service.ts

```diff
@@ -22,8 +20,9 @@ export class HistoryService {
   async saveStateTransition(
```

Comment:
do these need to be documented?

Reply:
Idk think they are pretty descriptive as is. Plus this shoulndt be used outside these use cases in the system.